pkgs/development/haskell-modules/patches/haddock-library-system-attoparsec.patch

```diff
@@ -0,0 +1,46 @@
+diff -Naur haddock-library-1.4.4-orig/haddock-library.cabal haddock-library-1.4.4/haddock-library.cabal
```

Comment:
Are those patch files maybe available for download from somewhere so that we don't have to check them into our repository?

Reply:
This one isn't available elsewhere. I should probably open an issue upstream, this is only needed because install doesn't work right yet with sublibraries.